### PR TITLE
28 GhostQuiz

### PR DESCRIPTION
**Pull request by mehtabkayani**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+BROWSER="C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"
+BROWSER_ARGS="--disable-web-security --disable-gpu --user-data-dir=~/chromeTemp"
```
Comment (JoakimAnder) [inline .env]: should be removed

File: .idea/workspace.xml [hunk 2]
```diff
@@ -2,16 +2,8 @@
 <project version="4">
```
Comment (JoakimAnder) [inline .idea/workspace.xml]: Why isn't this ignored? O.o

File: debug.log [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[0905/212738.228:ERROR:process_reader_win.cc(151)] SuspendThread: Access is denied. (0x5)
```
Comment (JoakimAnder) [inline debug.log]: debug log?? remove

File: src/App.js [hunk 4]
```diff
@@ -6,13 +6,25 @@ import Registration from "./Components/Registration";
...
+import GhostQuiz from "./Components/Game/QuizWindow/GhostQuiz";
 
 
 
 function App() {
-  const [page, setPage] = useState("");
+  const [page, setPage] = useState("game");
   const [token, setToken] = useState("");
-  const [user, setUser] = useState({});
+  const [user, setUser] = useState({attemptedLogins: 0,
```
Comment (JoakimAnder) [inline src/App.js]: nope

File: src/App.js [hunk 5]
```diff
@@ -22,11 +34,11 @@ function App() {
     setErrorMsgs(errorMsgs.concat(error))
   }
 
-
   function parseState(page) {
     switch (page) {
       case "profile": return <Profile setPage={setPage} user={user} setError={addError} setUser={setUser}/>;
       case "game": return <Game />;
+      // case "ghostQuiz": return <GhostQuiz/>
```
Comment (JoakimAnder) [inline src/App.js]: no ghostquiz here

File: src/App.js [hunk 6]
```diff
@@ -6,13 +6,25 @@ import Registration from "./Components/Registration";
 import Navbar from "./Components/Navbar";
 import ErrorMessage from "./Components/ErrorMessage";
 import Game from "./Components/Game";
+import GhostQuiz from "./Components/Game/QuizWindow/GhostQuiz";
```
Comment (JoakimAnder) [inline src/App.js]: remove ghostquiz import

File: src/Components/Game/QuizWindow/GhostQuiz/data.js [hunk 7]
```diff
@@ -0,0 +1,284 @@
+module.exports = {
```
Comment (JoakimAnder) [inline src/Components/Game/QuizWindow/GhostQuiz/data.js]: remove this entire file

File: src/Components/Game/QuizWindow/GhostQuiz/useFetch.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import {useEffect, useState} from 'react';
```
Comment (JoakimAnder) [inline src/Components/Game/QuizWindow/GhostQuiz/useFetch.js]: what is this?

File: src/Components/Game/QuizWindow/index.js [hunk 9]
```diff
@@ -1,18 +1,27 @@
 import React, {useState} from 'react';
+import { Button,Container, Row,Col,Card,CardDeck,CardGroup } from 'react-bootstrap';
+import GhostQuiz from "./GhostQuiz";
 
 const QuizWindow = () => {
     const [quiz, setQuiz] = useState("groupstage");
     function parseState() {
         switch(quiz) {
             case "test": return <div />;
+            case "ghostQuiz": return <GhostQuiz setQuiz={setQuiz}/>
```
Comment (JoakimAnder) [inline src/Components/Game/QuizWindow/index.js]: setquiz?

File: src/Components/Game/QuizWindow/index.js [hunk 10]
```diff
@@ -1,18 +1,27 @@
...
             {/*tabs*/}
+            <button>Group Stage</button>
+            <button onClick={()=>setQuiz("ghostQuiz")}>Ghost Quiz</button>
+            <button>Group Endings</button>
+            <button>Battle of the north</button>
             {/*quiz*/}
-            {parseState()}
+
+
+            {parseState(setQuiz)}
```
Comment (JoakimAnder) [inline src/Components/Game/QuizWindow/index.js]: parseState doesn't take a function as argument

File: .env (hunk 1 above)
Comment (mehtabkayani) [reply .env]: fixed

File: .idea/workspace.xml (hunk 2 above)
Comment (mehtabkayani) [reply .idea/workspace.xml]: ignored now

File: debug.log (hunk 3 above)
Comment (mehtabkayani) [reply debug.log]: removed

File: src/App.js (hunk 6 above)
Comment (mehtabkayani) [reply src/App.js]: removed

File: src/App.js (hunk 5 above)
Comment (mehtabkayani) [reply src/App.js]: fiexd

File: src/Components/Game/QuizWindow/GhostQuiz/data.js (hunk 7 above)
Comment (mehtabkayani) [reply src/Components/Game/QuizWindow/GhostQuiz/data.js]: removed

File: src/Components/Game/QuizWindow/GhostQuiz/useFetch.js (hunk 8 above)
Comment (mehtabkayani) [reply src/Components/Game/QuizWindow/GhostQuiz/useFetch.js]: removed

File: src/Components/Game/QuizWindow/index.js (hunk 9 above)
Comment (mehtabkayani) [reply src/Components/Game/QuizWindow/index.js]: removed

File: src/Components/Game/QuizWindow/index.js (hunk 10 above)
Comment (mehtabkayani) [reply src/Components/Game/QuizWindow/index.js]: removed

**Pull request merged**